### PR TITLE
Rename plus and minus to add and subtract

### PR DESCRIPTION
**Pull request by ptomato**
Closes: #770

### CODE REVIEW

**Comment by ptomato** [comment]
I'd recommend just skimming over the test262 files, it's way too much to review...

**Comment by Ms2ger** [comment]
Needs rebase

**Review by ryzokuken — APPROVED** [review]
Ignored test262 as you suggsted. Everything else looks great, just a few minor comments about... well, comments.

File: polyfill/lib/ecmascript.mjs [hunk 1]
```diff
@@ -1772,9 +1772,9 @@ export const ES = ObjectAssign({}, ES2019, {
         // convert weeks to days by calculating difference(relativeTo +
         //   { years, months }, relativeTo - { years, months, weeks })
         const yearsMonths = new TemporalDuration(years, months);
-        const yearsMonthsBefore = calendar.dateMinus(relativeTo, yearsMonths, {}, TemporalDate);
+        const yearsMonthsBefore = calendar.dateSubtract(relativeTo, yearsMonths, {}, TemporalDate);
```
[the remark this answers is not in the document]
Comment (ptomato) [reply polyfill/lib/ecmascript.mjs:1775]: Same here, this is fixed in #967.

**Pull request merged**